packager/mpd/base/mpd_utils.cc

```diff
@@ -162,6 +162,16 @@ std::string GetAdaptationSetKey(const MediaInfo& media_info,
   if (!ignore_codec) {
     key.append(":");
     key.append(GetBaseCodec(media_info));
+
+    if (media_info.video_info().has_transfer_characteristics()) {
```

Comment:
@sr1990, I added a comment to the Dolby section of this, and made it so that explicit transfer characteristics take priority over the assumption of PQ for Dolby.  Please take a look, and let me know if you agree.